includes/Modules/Idea_Hub.php

```diff
@@ -128,6 +159,96 @@ function( $maybe_empty, $postarr ) {
 		$this->post_topic_setting->register();
 	}
 
+	/**
+	 * Shows admin notification for idea hub ideas on post list screen.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $notices Array of admin notices.
+	 *
+	 * @return array Array of admin notices.
+	 */
+	private function googlesitekit_admin_notices( $notices ) {
+		if ( 'edit-post' !== get_current_screen()->id ) {
+			return $notices;
+		}
+
+		$notice_settings = array(
+			'saved' => array(
+				'text' => esc_html__( 'Need some inspiration? Revisit your saved ideas in Site Kit', 'google-site-kit' ),
+				'cta'  => esc_html__( 'See saved ideas', 'google-site-kit' ),
+				'link' => admin_url( 'admin.php?page=googlesitekit-dashboard#new-ideas' ),
+			),
+			'new'   => array(
+				'text' => esc_html__( 'Need some inspiration? Here are some new ideas from Site Kit’s Idea Hub', 'google-site-kit' ),
+				'cta'  => esc_html__( 'See new ideas', 'google-site-kit' ),
+				'link' => admin_url( 'admin.php?page=googlesitekit-dashboard#saved-ideas' ),
+			),
+		);
+
+		$dismissed_items = $this->dismissed_items->get_dismissed_items();
+		$ideas           = $this->transients->get( self::TRANSIENT_IDEAS );
+
+		if ( false === $ideas ) {
+			$ideas = array(
+				'saved-ideas' => $this->get_data( 'saved-ideas' ),
+				'new-ideas'   => $this->get_data( 'new-ideas' ),
+			);
+
+			$this->transients->set( self::TRANSIENT_IDEAS, $ideas, DAY_IN_SECONDS );
+		}
+		$dismissed_items = array();
```

Comment:
Looks like this is not needed, right? 
```suggestion
```

Reply:
https://github.com/google/site-kit-wp/pull/3668/commits/918df4e4ad848360545de51b12ac4167db594c2b